pkgs/development/python-modules/ring_doorbell/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, pytestCheckHook
+, pytz
+, requests
+, requests-mock
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "ring_doorbell";
+  version = "0.6.2";
```

Comment:
The latest release is [0.7.0](https://github.com/tchellomello/python-ring-doorbell/releases/tag/0.7.0).

Reply:
(See comment above, I initially tried this version but it doesn't appear to work with current latest Home Assistant)

Reply:
Home Assistant is pinning the releases. The problem is that bots could pick it up as outdated and then it will be updated anyways.

With my "Home Assistant hat" on:

I try to keep the requirements of Home Assistant as up-to-date as possible but I don't have access to every IoT device or service that is integrated. If you have a Ring device then please consider to update the [`ring` integration](https://github.com/home-assistant/core/blob/dev/homeassistant/components/ring/manifest.json#L5) to 0.7.0. This way all Ring users could benefit. Thanks.

Reply:
Looks like the issue is acknowledged as a bug upstream (https://github.com/tchellomello/python-ring-doorbell/issues/240), I don't think they intended a breaking change, so presumably it'll be fixed in a new release at some point.  From my point of view it seems best to stick on 0.6.2 until that fix comes out, rather than merge a known-buggy version.

I'm not very familiar with the upgrade bot process.  Assuming any changes are viewed by a human, perhaps a TODO comment iwould suffice?

Reply:
I think we should package version 7.0.0 and override the version inside of `pkgs/servers/home-assistant/default.nix`.

> Assuming any changes are viewed by a human, perhaps a TODO comment iwould suffice?

They are not, only in case of a build failure.

Reply:
I see, I'm not familiar with how this works so thanks for taking the time to explain!  I've made the suggested changes.